src/python/pants/backend/python/util_rules/pex_environment.py

```diff
@@ -159,15 +159,13 @@ async def find_pex_python(
     python_binary: PythonBinary,
     pex_runtime_env: PexRuntimeEnvironment,
     subprocess_env_vars: SubprocessEnvironmentVars,
-    global_options: GlobalOptions,
+    named_caches_dir: NamedCachesDirOption,
 ) -> PexEnvironment:
     return PexEnvironment(
         path=pex_runtime_env.path(python_bootstrap.environment),
         interpreter_search_paths=tuple(python_bootstrap.interpreter_search_paths()),
         subprocess_environment_dict=subprocess_env_vars.vars,
-        # TODO: This path normalization is duplicated with `engine_initializer.py`. How can we do
-        #  the normalization only once, via the options system?
```

Comment:
Removing this TODO is a lie. I didn't actually solve the problem, I only solved the problem of if any other call-sites will need the normalized path. We at most will duplicate 2 times, whereas before it would be N times.